Blockzilla/BrowserViewController.swift

```diff
@@ -538,6 +539,28 @@ class BrowserViewController: UIViewController {
 }
 
 extension BrowserViewController: URLBarDelegate {
+    
+    func urlBar(_ urlBar: URLBar, didAddCustomURL url: URL) {
+        // Add the URL to the autocomplete list:
+        let autocompleteSource = CustomCompletionSource()
+        
+        var absoluteString = url.absoluteString
+        if absoluteString.suffix(1) == "/" {
+            absoluteString = String(absoluteString.dropLast())
+        }
```

Comment:
What case is this code handling?

Reply:
Without dropping the last "/", the autocomplete URL's would end up having two of them. For example, if I quick added the URL "https://sfgov.org/" it would become "https://sfgov.org//" when auto-completing. Maybe there's a more elegant fix to this, though. 

[Here's an example](https://youtu.be/iP7M8qO94_0) of what it looks like without that line.

Reply:
thanks for the info, can you move this input sanitization to CustomCompletionSource.add(), and add a comment why this slash remover bit is there